returnn/datasets/util/vocabulary.py

```diff
@@ -165,9 +166,34 @@ def get_seq_labels(self, seq):
     :param list[int] seq:
     :rtype: str
     """
+    if isinstance(seq, int):
```

Comment:
Why is this needed? The documentation above says that it only accepts `list[int]`. Sth is wrong then.

But anyway, as commented elsewhere, I don't think you need any of the changes to `get_seq_labels`.